README.md

```diff
@@ -221,6 +221,20 @@ tensorflowjs_converter \
 
 The time of first call also includes the compilation time of WebGL shader programs for the model. After the first call the shader programs are cached, which makes the subsequent calls much faster. You can warm up the cache by calling the predict method with an all zero inputs, right after the completion of the model loading.
 
+6. I have models converted with previous version of TensorFlow.js converter (0.15.x), they contain model file in .pb extension, how do I convert them into new json output format?
+
+You can use the built-in migration tool to convert the models generated by previous versions. Here are the steps:
+
+```
+git clone git@github.com:tensorflow/tfjs-converter.git
+cd tfjs-converter
+yarn
+ts-node tools/pb2json_converter.ts pb_model_directory json_model_directory
```

Comment:
I don't think think this will work -- this assumes ts-node is globally installed. I think you need a `yarn` in front.

Also, to make it clear these are directories put a "/" at the end of each.